src/app/labels/labels.component.scss

```diff
@@ -0,0 +1,3 @@
+span {
+    margin-right: 12px;
```

Comment:
Added some spacing between the labels to allow us to see the full picture of what they look like on the style guide page.